modules/cnf-image-based-upgrade-prep.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/cnf-image-based-upgrade-base.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-prep_{context}"]
+= Moving to the Prep stage of the image-based upgrade with {lcao}
+
+When you deploy the {lcao} on a cluster, an `ImageBasedUpgrade` CR is automatically created.
```

Comment:
This section inside the generated doc seems like a duplicate.

`When you deploy the Lifecycle Agent on a cluster, an ImageBasedUpgrade CR is automatically created. You update this CR to specify the image repository of the seed image and to move through the different stages.

Moving to the Prep stage of the image-based upgrade with Lifecycle Agent
When you deploy the Lifecycle Agent on a cluster, an ImageBasedUpgrade CR is automatically created. You use this CR to specify the image repository of the seed image, refer to resources for backup and restore, and to move through the different stages.`